internal/peering-request-operator/broadcaster.go

```diff
@@ -41,6 +42,25 @@ func GetBroadcasterDeployment(request *discoveryv1alpha1.PeeringRequest, nameSA
 		remoteSA,
 	}
 
+	env := []v1.EnvVar{
```

Comment:
Could you please add some comments here to discuss why those variables have been added?